gsrs-spring-boot-autoconfigure/src/main/java/gsrs/controller/EtagLegacySearchEntityController.java

```diff
@@ -96,6 +97,12 @@
 
     @Autowired
     private GsrsExportConfiguration gsrsExportConfiguration;
+    
+//    public EtagLegacySearchEntityController() {super();}
+//    
+//    public EtagLegacySearchEntityController(ResultListRecordGenerator resultListRecordGenerator) {
+//    	super(resultListRecordGenerator);
+//    }
```

Comment:
Wiill this being commented out affect other things?
Or, can the commented out code be deleted?